docs/angular/api-nx-devkit/index.md

```diff
@@ -0,0 +1,1172 @@
+# Module: index
+
+## Table of contents
+
+### Enumerations
+
+- [ChangeType](/latest/angular/nx-devkit/index#changetype)
```

Comment:
These type of links: `/latest/angular/nx-devkit/index#changetype` will be problematic. Right now, we need to support 2 versions "_previous_" and "_latest_". As is, the release `current-1` ("_previous_") will still have `/latest/[...]` which will mess the internal routing for people looking at this release in particular.

We will also have a "preview" version for PRs, and the result will be the same. We definitely should find a solution to replace it with an interpolation marker like `{{version}}`.

Reply:
Ok, can you point me to where in the generators/executors api code it does interpolation replacement?  Or maybe give me a high-level view of how the api code is generated?  I could hook into your pipeline somewhere or just ditch typedoc and use the existing pipeline all the way

Reply:
Fixed by using a relative path